dm/dm/worker/subtask.go

```diff
@@ -665,6 +674,10 @@ func (st *SubTask) OperateSchema(ctx context.Context, req *pb.OperateWorkerSchem
 		return "", terror.ErrWorkerOperSyncUnitOnly.Generate(st.currUnit.Type())
 	}
 
+	if st.validatorStage() == pb.Stage_Running && req.Op != pb.SchemaOp_ListMigrateTargets {
+		return "", terror.ErrWorkerNotPausedStage.Generate(pb.Stage_Running.String())
```

Comment:
maybe add some hints about validator in this error

Reply:
https://github.com/pingcap/tiflow/pull/5025/commits/06b8febc7b53814756ea6edac66f173a36d090c1

Reply:
> I think we should only forbid operations that will change the schema.

yes, if subtask or validator is running we should  permit schema read operation, not write. i copied from previous branch, seems a bug. not sure if it affects v6.0, @Ehco1996 ptal.

Reply:
I think it is possible to allow read operations even when the task/validator is running,
but, in the [old logic](https://github.com/pingcap/dm/blob/63f574362d0aa1452e43eca3feab4a9ca2d32ee2/dm/worker/subtask.go#L570), OperateSchema was not allowed when the task was not pasued, we can improve this later so it not affects 6.0